src/main/java/htsjdk/samtools/cram/encoding/reader/MultiRefSeqIdReader.java

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************
+ * Copyright 2013 EMBL-EBI
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+package htsjdk.samtools.cram.encoding.reader;
+
+import htsjdk.samtools.ValidationStringency;
+import htsjdk.samtools.cram.io.BitInputStream;
+import htsjdk.samtools.cram.structure.*;
+
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A reader that only keeps track of alignment spans. The intended use is for
+ * CRAI index.
+ * 
+ * @author vadim
+ *
+ */
+public class MultiRefSeqIdReader extends CramRecordReader {
```

Comment:
Renamed with "Multi" to be more explicit about when it's used

Reply:
@cmnbroad I actually had an earlier version of the code which did exactly that.  However it relied on the `APDelta` assumption which we discussed in Slack recently.  I still have that branch and we can revisit this plan after #1218 is involved.

Reply:
Updated to use new name and added Slice method